inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/util/StringUtil.java

```diff
@@ -106,4 +245,26 @@ public static String formatDate(Date date) {
         return sdf.format(date);
     }
 
+    public static long parseDateTime(String value) {
+        try {
+            if (value.length() < 8) {
+                return -1;
+            } else if (value.length() <= 9) {
+                SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyyMMdd");
+                Date date = simpleDateFormat.parse(value.substring(0, 8));
+                return new Timestamp(date.getTime()).getTime();
+            } else if (value.length() <= 11) {
+                SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyyMMddHH");
+                Date date = simpleDateFormat.parse(value.substring(0, 10));
+                return new Timestamp(date.getTime()).getTime();
+            } else {
+                SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyyMMddHHmm");
+                Date date = simpleDateFormat.parse(value.substring(0, 12));
+                return new Timestamp(date.getTime()).getTime();
+            }
+        } catch (ParseException e) {
+            throw new IllegalArgumentException("Unexpected time format : " + value + ".");
```

Comment:
Not need append `.` at the end of exception message.

Reply:
fixed, thx